app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -52,7 +53,7 @@ private class NodeData(
   val items: Map<String, RegistryItem> = emptyMap()
 ) {
 
-  constructor(creator: () -> Story): this({ _ -> creator() })
+  constructor(creator: () -> Story) : this({ _ -> creator() })
```

Comment:
Why this change? All other entries in this file don't have the space before.

Reply:
The auto-formatter does that only for this instance. Maybe it has a different rule for constructors.